backend/auth-service/run.sh

```diff
@@ -0,0 +1,9 @@
+if grep -q "otel" <<< "$TRACING_PROFILE"; then
```

Comment:
We will have one environment and one SDKs source which is OTel. So no need to use this run.sh file